student-test/assignment1/Test_validate_username.c

```diff
@@ -14,9 +14,5 @@
 */
 void test_validate_my_username()
 {
-    /**
-     * TODO: Replace the line below with your code here as described above to verify your /conf/username.txt 
-     * config file and my_username() functions are setup properly
-     */
-    TEST_ASSERT_TRUE_MESSAGE(false,"AESD students, please fix me!");
+    TEST_ASSERT_EQUAL_STRING_MESSAGE(my_username(), malloc_username_from_conf_file(), "Username mismatch!");
```

Comment:
@krish0706 what happens to the string returned by `malloc_username_from_conf_file` here?